resources/grch38/reference_data.py

```diff
@@ -0,0 +1,59 @@
+from resources.resource_utils import *
+
+# Resources with no versioning needed
+purcell_5k = TableResource(
+    path='gs://gnomad-public/resources/grch38/purcell5k.ht',
+    source_path="gs://gnomad-public/intervals/purcell5k.interval_list"
+)
+
+na12878_giab = MatrixTableResource(
+    path='gs://gnomad-public/truth-sets/hail-0.2/HG001_GRCh38_GIAB_highconf_CG-IllFB-IllGATKHC-Ion-10X-SOLID_CHROM1-X_v.3.3.2_highconf_PGandRTGphasetransfer.mt',
+    source_path='gs://gnomad-public/truth-sets/source/HG001_GRCh38_GIAB_highconf_CG-IllFB-IllGATKHC-Ion-10X-SOLID_CHROM1-X_v.3.3.2_highconf_PGandRTGphasetransfer.vcf.gz'
+)
+
+# Versioned resources: versions should be listed from most recent to oldest
+clinvar = VersionedTableResource(
+    default_version='20190923',
+    versions={
+        '20190923': TableResource(
+            path="gs://gnomad-public/resources/grch38/clinvar_20190923.ht",
+            source_path='gs://gnomad-public/resources/grch38/clinvar_20190923.vcf.gz'
+        )
+    }
+)
+
+dbsnp = VersionedTableResource(
+    default_version='b151',
+    versions={
+        'b151': TableResource(
+            path='gs://gnomad-public/resources/grch38/dbsnp_b151_grch38_all_20180418.ht',
+            source_path='gs://gnomad-public/resources/grch38/dbsnp_b151_grch38_all_20180418.vcf.bgz',
+            attributes={'vcf_header_path': 'gs://gnomad-public/resources/grch38/dbsnp_b151_grch38_all_20180418.vcf.header'}
+        )
+    }
+)
+
+hapmap = TableResource(
```

Comment:
Should  `hap_map`,`kgp_omni`, and `mills` be moved above the versioned resources? Or if not and they are versioned, why dont they have a version?

Reply:
they could, but does the order really matter?